### ISSUE TITLE
Issues around direct publishing

### ISSUE THREAD

**Issue by FynnBe** [bioimage-io/bioimageio-uploader-service#11]
There was a recent change that a reviewer's `accept` action does not dispatch the `publish` collection workflow, but renames the file on S3 directly instead.
@oeway , I know we talked about it, and yes the workflow mainly only renames the folder, but I gave it some more thought and would argue that we should definitely go via the workflow dispatch again because:

1. single source of truth; if anything in the publish process changes we have to keep these two avenues in sync
2. they have not been in sync so far as there are these differences: the collection workflow...
    * updates the collection.json which is ultimately what is shown on the website
    * writes to the log (updates the log files with the `accepted` status and then the `published` status)
    * afterwards triggers the `backup` workflow to backup to zenodo and generate a DOI (this is also done once a week on a schedule, so this is just causing a delay to get the DOI for the published wf)
    these changes can of course be fixed, but I want to stress again that this needs fixing only because we have issue number 1.
3. I don't think a 3 min delay from a reviewer accepting a resource to it being published matter all too much. Given that the uploader will typically have to wait for the review longer than x minutes anyway...

**Comment by oeway** [comment]
I agree that we should always go via a workflow dispatch. Also, it doesn't really matter, whatever way we decided please just update the bioimageio_collection_backoffice, the backend just [call the publish function](https://github.com/bioimage-io/bioimageio-uploader-service/blob/2a334a68ba57daae6080a9b86905495a95b0eb03/bioimageio_uploader_service/api.py#L71C24-L75) anyway.
